build/includes/google-cloud.mk

```diff
@@ -39,27 +39,27 @@ clean-gcloud-test-cluster: $(ensure-build-image)
 gcloud-e2e-test-cluster: GCP_PROJECT ?= $(current_project)
 gcloud-e2e-test-cluster: $(ensure-build-image)
 gcloud-e2e-test-cluster:
-	$(MAKE) terraform-init DIRECTORY=e2e
+	$(MAKE) terraform-init TERRAFORM_BUILD_DIR=$(mount_path)/build/terraform/e2e
```

Comment:
How did this work in the first place?

Reply:
Sorry for not touching this for so long. Returning back to this PR.